### DISCUSSION TITLE
cmake build error

### DISCUSSION THREAD

**Discussion by woshiHhnecX** [o3de/o3de#5156] [Help Wanted] [answered]
I tried it on my colleague's computer. His computer doesn't have this problem.Since I have not used VS2019 before, 
I do not know if it is the configuration problem of VS2019. The two of us worked on it for a long time and didn't know why
someone can tell us the specific reason and the solution, thanks!!!

steps：
1. cmake -B ./build -S ./o3de -G "Visual Studio 16" -DLY_3RDPARTY_PATH=./o3de-packages     (SUCCEED)
2. scripts\o3de.bat register --this-engine         (SUCCEED)
3. scripts\o3de.bat create-project --project-path ../project     (SUCCEED)
4. cmake -B ./build_project -S ./project -G "Visual Studio 16" -DLY_3RDPARTY_PATH=./o3de-packages    (SUCCEED)
5. cmake --build ./build_project --target project.GameLauncher Editor --config profile -- /m    (ERROR)

the step 5 is error and error message:
D:\o3de\Code\Framework\AzCore\AzCore/JSON/rapidjson.h(54,10): fatal error C1083: can not open file: “rapidjson/rapidjson.h”: No such file or directory [D:\build_proj
ect\o3de\Code\Framework\AzCore\AzCore.vcxproj]

environment：
windows 10 ; vs2019 ; cmake  3.22.0-rc2

**Comment by woshiHhnecX** [comment, marked as answer]
over.  change cmake version 3.20.6

**Comment by amznestebanpapp** [comment]
https://github.com/o3de/o3de/pull/5314 addressed CMake 3.22.0-rc2
